doc/scapy/installation.rst

```diff
@@ -191,7 +191,7 @@ Here are the topics involved and some examples that you can use to try if your i
      >>> dec.show()
      >>> dec[0]
  
-* PKI operations and TLS decryption. `cryptography <https://cryptography.io>` is also needed.
+* PKI operations and TLS decryption. `cryptography <https://cryptography.io>`_ is also needed.
```

Comment:
What are you changing here?

Reply:
This is the proper format: as you can see, the link is broken in current doc http://scapy.readthedocs.io/en/latest/installation.html#optional-software-for-special-features